dev.py

```diff
@@ -46,6 +46,29 @@ def cmd_all_npm_build(args):
         frontend_dir = next(project_dir.glob("*/frontend/"))
         run_verbose(["npm", "run", "build"], cwd=str(frontend_dir))
 
+
+def cmd_all_install_python_deps(args):
+    """"Install all dependencies needed to run e2e tests for all examples and templates"""
+    for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
+        run_verbose(["pip", "install", "-e", ".[devel]"], cwd=str(project_dir))
+
+
+def cmd_all_install_browsers(args):
+    """"Install multiple browsers to run e2e for all examples and templates"""
+    for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
+        e2e_dir = next(project_dir.glob("**/e2e/"), None)
+        if e2e_dir:
+            run_verbose(["playwright", "install", "webkit", "chromium", "firefox", "--with-deps"], cwd=str(project_dir))
```

Comment:
Do we have to install the browser for each package separately?

Reply:
I'm not sure. We install playwright for each package separately so I thought that we should install browsers the same way, but I will try to install them once

Reply:
Ok, it worked :)